docs/tutorials/fundamentals/tutorial-token-transfer-and-nft-purchase.md

```diff
@@ -0,0 +1,136 @@
+
+![](./images/token-transfer-ultra-wallet-confirm.png)
+
+### Step 6: Transaction Completion
+
+After signing the transaction, you will see a confirmation screen indicating that the transaction was successfully completed.
+
+![](./images/token-transfer-txn-completed.png)
+
+### Step 7: Verify the Transaction
+
+To ensure the transaction was successfully processed, visit the Ultra Testnet Explorer at `https://explorer.testnet.ultra.io/`. Search for your account (`1aa2aa3aa4in`) to view the transaction details and confirm that the transfer went through as intended.
+
+![](./images/token-transfer-explorer.png)
+
+## Purchasing Digital Assets (Uniqs)
+
+To purchase a Uniq from the Token Factory using the Ultra toolkit, follow these steps carefully:
+
+### Step 1: Verify Factory Details
+
+Begin by checking the Factory Explorer in the Uniq Explorer. Confirm the on-chain data for the Token Factory you are interested in, which in this case is Factory ID 4243, owned by account `1aa2aa3aa4io`.
+
+![](./images/purchase-factory-explorer.png)
+
+### Step 2: Explore Available Uniqs
+
+View the Uniqs available under Token Factory 4243. This screen will show you the specific Uniqs that you can purchase from this factory.
+
+![](./images/purchase-factory-explorer-uniqs.png)
+
+### Step 3: Initiate Purchase Action
+
+Navigate to the action builder in your toolkit and select the `eosio.nft.ft` contract. Choose the purchase action to start the process of buying a Uniq.
+
+![](./images/purchase-action-builder.png)
+
+### Step 4: Enter Purchase Details
+
+On the purchase action screen, fill out the attributes necessary for the transaction:
+- **Factory ID**: `4243` (ID of the Token Factory)
+- **Maximum Price**: `1.00000000 UOS` (maximum amount you are willing to pay)
+- **Buyer**: `1aa2aa3aa4in` (your account)
+- **Receiver**: `1aa2aa3aa4in` (your account, receiving the Uniq)
+
+![](./images/purchase-attribute-I.png)
+
+Scroll down to complete any additional required attributes for the purchase.
+
+![](./images/purchase-attribute-II.png)
+
+### Step 5: Review and Confirm Transaction
+
+Carefully review all the transaction details on the confirmation screen to ensure they are correct before proceeding.
+
+![](./images/purchase-confirm.png)
+
+### Step 6: Confirm and Sign with Ultra Wallet
+
+Confirm and sign the transaction using the Ultra Wallet extension. This step is essential to authorize the transaction on the blockchain.
+
+![](./images/purchase-confirm-ultra-wallet.png)
+
+### Step 7: Transaction Completion
+
+Once the transaction is signed, a confirmation screen will display indicating that the purchase was successfully completed.
+
+![](./images/purchase-ultra-wallet-confirm.png)
+
+### Step 8: Verify Ownership of New Uniq
+
+To confirm the purchase, revisit the Factory Explorer page for Factory 4243. Verify that the new Uniq now appears under your account `1aa2aa3aa4in`.
```

Comment:
Better to direct the user in this case to this page instead: https://toolkit.ultra.io/user. Here they can just enter their account name. Or ask them to go to https://toolkit.ultra.io/user and when they are logged in the first button will be `Inventory`.

The reason is that there will be potentially many owners of the Uniqs from this factory and you will need to go through multiple pages until you get to your account. In the User page it will be much clearer since it is only a single Uniq that will be listed.

Reply:
Thanks. Modified this step.